qiskit_experiments/library/characterization/t1_analysis.py

```diff
@@ -159,6 +159,7 @@ def _format_plot(cls, ax, analysis_result, qubit=None, add_label=True):
         if qubit is not None:
             ax.set_title(f"Qubit {qubit}", fontsize=16)
         ax.set_xlabel("Delay (s)", fontsize=16)
+        ax.ticklabel_format(axis="x", style="sci", scilimits=(0, 0))
```

Comment:
```suggestion
        ax.ticklabel_format(axis="x", style="sci", scilimits=(-3, 3))
```
`scilimits=(-3,3)` ought to be enough.